docs/cluster_spec.md

```diff
@@ -816,6 +816,20 @@ spec:
 
 Note that Kubelet will fail to install the shutdown inhibtor on systems where logind is configured with an `InhibitDelayMaxSeconds` lower than `shutdownGracePeriod`. On Ubuntu, this setting is 30 seconds.
 
+### SeccompDefault
+
+[SeccompDefault](https://kubernetes.io/blog/2021/08/25/seccomp-default/) enables the use of `RuntimeDefault` as the default seccomp profile for all workloads. (Default: false)
+
+Note that a feature gate is required to enable the feature, and the feature is turned on using kubelet config.
+
+```yaml
+spec:
+  kubelet:
+    featureGates:
+      SeccompDefault: "true"
+    seccompDefault: true
```

Comment:
This feels a bit unnecessary. Would it make sense to just automatically enable the feature flag when `seccompDefault: true`?

Reply:
I went ahead and pushed a commit for this change. Happy to revert if there is a preferred way to handle this type of configuration.

Reply:
Thanks for the thought, but it is not the direction we want to go. We expect operators to set these flags and maintain them.